packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -0,0 +1,103 @@
+<h1 data-translate="datacenter_backup_title"></h1>
+<div class="datacenter-backup" data-ui-view>
+    <div data-ovh-alert="{{ $ctrl.alertMain }}" id="backupMainAlert"></div>
+    <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
```

Comment:
A static model ?

Reply:
This is not completely implemented. Its a part of https://github.com/ovh/manager/pull/2350
:)

Reply:
The PR has been merged and this is complete now